app/views/obituaries/show.html.erb

```diff
@@ -1,14 +1,12 @@
 <!-- Page Content -->
 <div class="container", style= "background-color:white;">
 
-  <!-- Portfolio Item Heading -->
   <h1 class="my-4"><%="#{@obituary.name}"%></h1>
 
-  <!-- Portfolio Item Row -->
   <div class="row" style="padding-bottom: 50px;">
 
     <div class="col-md-4" id="obituary-image">
-      <%= image_tag(@obituary.image_url) rescue image_tag 'https://image.shutterstock.com/image-vector/ui-image-placeholder-wireframes-apps-260nw-1037719204.jpg', {class: "img-fluid"} %>
+      <%= image_tag(@obituary.image_url, width: 300) rescue image_tag 'https://image.shutterstock.com/image-vector/ui-image-placeholder-wireframes-apps-260nw-1037719204.jpg', {class: "img-fluid"} %>
```

Comment:
Image on show page can be a little bigger, but didn't want to make it too big in case the photos were small and they got all pixilated and weird.